KomaMRICore/src/simulation/Bloch/BlochSimulationMethod.jl

```diff
@@ -2,7 +2,7 @@ struct Bloch <: SimulationMethod end
 
 export Bloch
 
-include("Magnetization.jl") #Defines Mag <: SpinStateRepresentation
+#include("Magnetization.jl") #Defines Mag <: SpinStateRepresentation
```

Comment:
Why did you do this??

Reply:
Now is included in the file of the definition of the KomaMRICore module. Addressed in https://github.com/cncastillo/KomaMRI.jl/pull/240/commits/8cf10212c4249fd8944bf303f0330a03dd071cca#diff-32a3a886ff90d041566b94510fc6fddf92fbf30bbfa142c30537d2b761a7ca68